updates/stable.json

```diff
@@ -13,19 +13,19 @@
       }
     },
     {
-      "version": "32.20200809.3.0",
+      "version": "32.20200824.3.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "32.20200824.3.0",
+      "version": "32.20200907.3.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 2880,
-          "start_epoch": 1599579000,
+          "start_epoch": 1569249000,
```

Comment:
> stable rollout of 32.20200907.3.0 scheduled for 2019-09-23 14:30:00 UTC for 2880m (48.0h)

Reply:
I should clearly get more coffee before doing this kind of reviews, as I failed to notice in multiple spots that this has **2019** as start year :disappointed:.